mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -1335,20 +1370,26 @@ inline void computeTopAndBottomIndicesWithAffineMap(
 //===----------------------------------------------------------------------===//
 
 // The ArgumentFields keep track of differences between conv operations
```

Comment:
this comment wrt `ArgumentFields` should be moved or eliminated along with `ArgumentFields`.

Reply:
done